src/plotrecipes/polygons.jl

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License. See LICENSE in the project root.
 # ------------------------------------------------------------------
 
-@recipe function f(polygon::Polygon)
+@recipe function f(polygon::PolySurface)
```

Comment:
Missed this one. I would do a /search/replace in the codebase to update every instance, including the tests.